addons/website/static/tests/tours/rte.js

```diff
@@ -115,6 +115,7 @@ wTourUtils.dragNDrop({
 }, {
     content: "check if translation is activate",
     trigger: 'iframe [data-oe-translation-initial-sha]',
+    isCheck: true,
```

Comment:
why this change ?

Reply:
This step is not supposed to click on the `[data-oe-translation-initial-sha]`, it's only supposed to check it's existing. As you can see, the next step is typing text in a `p` in the `wrap`. So the attribute translation dialog is not supposed to open.